script/Ciel/Day 7/07_01_CIEL07_4.txt

```diff
@@ -149,20 +162,21 @@ When she puts it like that, there's not much room for a rebuttal.
 [sha:3ad505acfea25a23a54abdcc2b4645bf0357bc69]{
 -- Page 410, Offset 15793.
 -- 俺のような自由人が家に帰ってこないのなら、その理由は遊び以外の何ものでもないからだ。
-If someone with as much free-time as I have isn't returning home early,
- then there can't be any other reason than that I'm out fooling around.
+If someone with as few commitments as myself isn't returning home early,
+ there isn't much they could be off doing other than fooling around.
 }
 [sha:50bef6066fc91e27d01e58fb70da6bcaf6ee1d7a]{
 -- Page 411, Offset 15794.
 -- 「もっとも、門限を守ればいい、という話ではありませんけど。これまでの自堕落ぶりを反省して早く帰ってきてくれたのは喜ばしいですが、その後が気になります。夕食から心ここにあらずという様子ですが、何かあったのですか？
-"Although, that doesn't mean all you have to do is follow the curfew.
- I am pleased that you've reflected on your self-indulgence up until now and returned home early, but I'm still concerned about what comes thereafter.
- You appear to have had your mind elsewhere ever since dinner――did something happen?"
+"Of course, adhering to the curfew isn't the only thing I expect of you.
```

Comment:
I feel like 'Although' conveys the meaning better here.

Reply:
"Of course" holds the meaning imo and sounds way more akiha-like